destination/format/csv.go

```diff
@@ -520,12 +532,10 @@ func mapAvroToSnowflake(ctx context.Context, field *avro.Field) (string, error)
 
 	// fixed types
 	f, ok := t.(*avro.FixedSchema)
-	if ok {
-		switch f.Logical().Type() {
-		case avro.Decimal:
-			fmt.Println("decimal detected")
-			return SnowflakeFloat, nil
-		}
+	if ok && f.Logical().Type() == avro.Decimal {
+		fmt.Println("decimal detected")
```

Comment:
Let's use the logger here. The log level is debatable, technically it should be trace, otherwise we'll see _a ton_ of these messages, every time we process a record.
```suggestion
		sdk.Logger(ctx).Trace().Msg("decimal detected")
```